tests/foreman/api/test_reporttemplates.py

```diff
@@ -38,6 +39,41 @@
 from robottelo.vm import VirtualMachine
 
 
+@pytest.fixture(scope='class')
+def setup_content(request):
```

Comment:
On https://github.com/SatelliteQE/robottelo/pull/7651/files it is called `content_setup` here it is `setup_content` any reason for different names?

Reply:
Also in future we might want to revisit this fixtures and put a single dynamic one in `tests/conftest.py` so we avoid repetition.


![same_code](https://user-images.githubusercontent.com/458654/75479013-a1e2c980-597d-11ea-8999-fb8afd469262.png)

Reply:
Figures out that we already have a standard for reusable fixtures, examples here: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/conftest.py#L8-L27

Reply:
I will make a note for future tests that require a full content setup.